testing-modules/junit-5/src/test/java/com/baeldung/junit5/testinstance/ReportSerializerUnitTest.java

```diff
@@ -0,0 +1,66 @@
+package com.baeldung.junit5.testinstance;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+
+@TestInstance(Lifecycle.PER_CLASS)
+public class ReportSerializerUnitTest {
+
+    private String largeContent;
+    private String content = "Lorem ipsum dolor sit amet, eam no tale solet patrioque, est " + "ne dico veri. Copiosae petentium no eum, has at wisi dicunt causae. Duo ea ";
+    private String smallContent;
+
+    private Report report;
+
+    @BeforeAll
+    void setUpFixture() {
+
+        StringBuilder reportBuilder = new StringBuilder();
+        smallContent = "Lorem ipsum dolor";
+
+        reportBuilder = new StringBuilder();
+        for (int i = 0; i < 100000; i++) {
```

Comment:
I think this will create a lot of work for our test builds... can you replace `100000` with `LARGE_NUMBER` and then define that constant as `2` with a comment next to it saying `// make this large, like 100000 to demonstrate the slow down`